<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PublishPoundageWidget</class>
 <widget class="QWidget" name="PublishPoundageWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>770</width>
    <height>530</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>95</y>
     <width>670</width>
     <height>290</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{font: 11px &quot;微软雅黑&quot;;color:rgb(52,37,90);}
#widget{background-color: rgb(255, 255, 255);border-radius:15px;}</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>15</y>
      <width>200</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12px &quot;微软雅黑&quot;;
color:rgb(83,107,215);</string>
    </property>
    <property name="text">
     <string>发布账号：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>75</y>
      <width>200</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12px &quot;微软雅黑&quot;;
color:rgb(83,107,215);</string>
    </property>
    <property name="text">
     <string>承兑币种:</string>
    </property>
   </widget>
   <widget class="QToolButton" name="cancelBtn">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>230</y>
      <width>96</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>取消</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_fee">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>160</y>
      <width>251</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>-1</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12px &quot;微软雅黑&quot;;
color:rgb(83,107,215);</string>
    </property>
    <property name="text">
     <string>20 XWC</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>75</y>
      <width>200</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12px &quot;微软雅黑&quot;;
color:rgb(83,107,215);</string>
    </property>
    <property name="text">
     <string>XWC 数量：</string>
    </property>
   </widget>
   <widget class="QToolButton" name="okBtn">
    <property name="geometry">
     <rect>
      <x>225</x>
      <y>230</y>
      <width>96</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>确定</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_targetType">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>105</y>
      <width>251</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>135</y>
      <width>200</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12px &quot;微软雅黑&quot;;
color:rgb(83,107,215);</string>
    </property>
    <property name="text">
     <string>手续费:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_source">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>105</y>
      <width>251</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>15</y>
      <width>200</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12px &quot;微软雅黑&quot;;
color:rgb(83,107,215);</string>
    </property>
    <property name="text">
     <string>承兑数量:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_target">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>44</y>
      <width>251</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="Line" name="line_6">
    <property name="geometry">
     <rect>
      <x>326</x>
      <y>24</y>
      <width>20</width>
      <height>171</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_accounts">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>44</y>
      <width>251</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="title_label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>28</y>
     <width>200</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>发布承兑单</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
